examples/stupid-s.mkd

```diff
@@ -2,7 +2,6 @@
 title: STUN/TURN using PHP in Despair
 abbrev: STuPiD
 docname: draft-hartke-xmpp-stupid-00
-date: 2009-07-05
```

Comment:
Why is that change desirable?

Reply:
Because anyone pulling this down today will get an error when they try to build because the date is too far back.

Reply:
Wow.  Need to file a bug report.
(The file was supposed to produce an exact copy of https://datatracker.ietf.org/doc/draft-hartke-xmpp-stupid/ .)

Reply:
```
xml2rfc  stupid-s.xml
/Volumes/nar/Users/cabo-rescue/reo/src/kramdown-rfc2629/examples/stupid-s.xml(42): Warning: The document date (2009-07-05) is more than 3 days away from today's date
/Volumes/nar/Users/cabo-rescue/reo/src/kramdown-rfc2629/examples/stupid-s.xml(706): Warning: Artwork too wide, reducing indentation from 0 to 0
/Volumes/nar/Users/cabo-rescue/reo/src/kramdown-rfc2629/examples/stupid-s.xml(706): Warning: Figure too wide, reducing indentation from 3 to 0
/Volumes/nar/Users/cabo-rescue/reo/src/kramdown-rfc2629/examples/stupid-s.xml(704): Warning: Section too wide, reducing indentation from 0 to 0
/Volumes/nar/Users/cabo-rescue/reo/src/kramdown-rfc2629/examples/stupid-s.xml(335): Warning: Back too wide, reducing indentation from 0 to 0
/Volumes/nar/Users/cabo-rescue/reo/src/kramdown-rfc2629/examples/stupid-s.xml(706): Warning: Too long line found (L650), 4 characters longer than 72 characters: 
                  "WHERE `timestamp` < DATE_SUB(NOW(), INTERVAL 5 MINUTE)");
 Created file stupid-s.txt
```

A couple of warnings (some rather interesting), but no error.